### PR TITLE
test(jiva-volume): add unit tests for jiva volumes

### PR DESCRIPTION
**Pull request by payes**
This PR adds the following tests to the suite:
1. Volume Expansion
2. Volume Creation/Deletion without app deployment
3. Remount
Signed-off-by: Payes Anand <payes.anand@mayadata.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ before_install:
   - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.6.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
   - mkdir -p $HOME/.kube $HOME/.minikube
   - touch $KUBECONFIG
-  - sudo minikube start --vm-driver=none --kubernetes-version=v1.15.7
+  - sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=v1.18.1
```
Comment (prateekpandey14) [inline .travis.yml:41]: We need to add the `before_install` step to install the conntrack pkg to use the latest release of k8s
```
before_install:
- sudo apt-get -qq -y install conntrack
```
Check here
https://github.com/openebs/cstor-csi/blob/b82d38349120403ebb63973d045ccac3cf16b383/.travis.yml#L20

File: .travis.yml [hunk 2]
```diff
@@ -37,7 +37,8 @@ before_install:
   - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.6.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```
Comment (prateekpandey14) [inline .travis.yml]: minikube version v1.8.1

Comment (prateekpandey14) [reply .travis.yml]: kubectl version v1.18.1

File: tests/remount_test.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	})
+})
+
+func remountTest() {
+	By("creating and verifying PVC bound status", createAndVerifyPVC)
+	By("Creating and deploying app pod", createDeployVerifyApp)
+	By("scale down jiva controller pod", scaleDownControllerPod)
+	By("Verify app state as CrashLoopBackOff", func() { verifyCrashLoopBackOffStateOfAppPod(true) })
+	By("Scale up jiva controller pod", scaleUpControllerPod)
+	By("Verify app state as not CrashLoopBackOff", func() { verifyCrashLoopBackOffStateOfAppPod(false) })
```
Comment (prateekpandey14) [inline tests/remount_test.go:38]: nit : `Verify app state as Running` ?

Comment (prateekpandey14) [inline tests/remount_test.go:38]:
```suggestion
	By("Verify app state as Running", func() { verifyCrashLoopBackOffStateOfAppPod(false) })
```

Comment (payes) [reply tests/remount_test.go:38]: Actually, if you look at the pod yaml, the status.phase feild always remains as Running even though the pod is in CrashLoopBackOff. That is the reason I used the above log.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**